event/listener.php

```diff
@@ -118,23 +115,13 @@ public function show_page_links()
 				continue;
 			}
 
-			// Assign any available custom icons for the current link in the user's style
-			$custom_icon = '';
-			foreach ($pages_icons as $icon_path => $ext_name)
-			{
-				if (strpos($icon_path, $this->user->style['style_path'] . '/theme/images/pages_' . $row['page_route'] . '.gif') !== false)
-				{
-					$custom_icon = 'pages_' . $row['page_route'] . '.gif';
-					break;
-				}
-			}
-
 			// Assign template var data
 			$this->template->assign_block_vars($row['page_link_event_name'] . '_links', array(
 				'U_LINK_URL' => $this->helper->route('phpbb_pages_dynamic_route_' . $row['page_id']),
 				'LINK_ROUTE' => $row['page_route'],
 				'LINK_TITLE' => $row['page_title'],
-				'ICON_LINK' => $custom_icon,
+				'ICON_FONT'  => $row['page_icon_font'],
+				'ICON_LINK'  => !$row['page_icon_font'] ? $this->page_operator->get_page_icon($row['page_route']) : '',
```

Comment:
I also like shorthand for `!= ''`, but I remember from the old times that in my MODs I was instructed to use `!empty()` instead. Maybe this is the case as well?